__model/Reviews.model.js

```diff
@@ -26,11 +26,38 @@ exports.SelectReviews = (order = "DESC" ) => {
       });
   };
   
-  exports.SelectReviewById = (review_id) => {
+  exports.SelectReviewById = (rev_id) => {
     return db
       .query("SELECT * FROM reviews WHERE review_id = $1;", [review_id])
       .then((result) => {
         if(result.rows.length === 0){
           return Promise.reject({ status: 404, msg: "ID not found !" });}
         return result.rows[0]});
+  }
+  exports.patchComment = (rev_id,votes) => {
+    const {inc_votes} = votes
+    return checkIDExists(rev_id).then(() => {
+   if(inc_votes > 0)
+   {
+    return db
+    .query(`UPDATE reviews 
+              SET votes = votes + ${inc_votes}
+              WHERE review_id = ${rev_id}
+              RETURNING * `)
+    .then((result) => {
+        return result.rows;
+      });
+   }
+   else
+   {
+    return db
+    .query(`UPDATE reviews 
+              SET votes = votes  ${inc_votes}
+              WHERE review_id =  ${rev_id}
+              RETURNING * `)
+    .then((result) => {
+        return result.rows;
+      });
+   }
+  })
```

Comment:
I imagine you have done this because of negative integers, but you actually dont need to; if you do votes + -5, it will just minus 5 from the votes so you can use one query for both possibilities